js/components/graph/Graph.js

```diff
@@ -11,7 +11,6 @@ import NodeGroup from "./NodeGroup"
 import GraphDropdown from "./GraphDropdown"
 import Sidebar from "./Sidebar"
 import { parseAnd } from "../../util/util.js"
-import { refLookUp } from "../common/utils"
```

Comment:
Good! It looks like this function is no longer being used, so you can delete it altogether.